woocommerce/woocommerce-gutenberg-products-block
# Babel and Autoprefixer Dependency Updates

The autoprefixer dependency update broke builds do to some changes in the peer dependencies. To resolve this, I've updated our Babel dependencies too, and updated the webpack build to include proposals (such as optional chaining) that we're using.

See https://github.com/woocommerce/woocommerce-blocks/pull/8794

### Testing

1. Confirm dev build completes `nvm use && npm install && npm run dev`
2. Confirm production build completes `nvm use && npm install && npm run build`
3. Smoke test some blocks/e2e should run and pass
